.github/workflows/release_notes_check.yml

```
@@ -0,0 +1,28 @@
+name: Release Notes Check
+
+on:
+  pull_request:
+    types:
+      # On by default if you specify no types.
+      - "opened"
+      - "reopened"
+      - "synchronize"
+      # For `skip-label` only.
+      - "labeled"
+      - "unlabeled"
+
+jobs:
+  check_release_notes:
```

```suggestion
  check-release-notes:
```

Just to be consistent with the style in other workflows.